.github/actions/paths-filter/action.yml

```diff
@@ -183,6 +183,18 @@ outputs:
         steps.filter-common.outputs.maven-change == 'true' ||
         steps.filter-common.outputs.rdbms-change == 'true'
       }}
+  zeebe-base-module-changes:
+    description: Output whether backend tests should be run if base level modules or Zeebe have changed
+    value: >-
+      ${{
+        github.event_name == 'push' ||
```

Comment:
not sure I understand the idea here - this means Zeebe tests will be run on every commit pushed to a branch?

Reply:
I'm probbably reading this wrong, but since it's a chain of local OR calls, if `github.event_name == 'push'`, regardless of anything else, `zeebe-base-module-changes` will be set to true?